webapp/src/landing-page/components/MenuMainItem/index.tsx

```diff
@@ -1,21 +1,14 @@
 import React from 'react';
 
-import { StyledLink } from './style';
+import { StyledLink } from './StyledLink';
 
 export interface IProps {
   children: string;
   to: string;
 }
 
-const MenuMainItem = ({ to, children }: IProps) => (
-  <StyledLink
-    {...{
-      activeClassName: 'active',
-      to,
-    }}
-  >
+export const MenuMainItem = ({ to, children }: IProps) => (
+  <StyledLink {...{ to }}>
```

Comment:
Зачем такие извращения? :)

```suggestion
  <StyledLink to={to}>
```

Reply:
Тут была матня ещё с именем класса, я её снёс

Reply:
> Тут была матня ещё с именем класса, я её снёс

Ок, но лучше поправить

Reply:
Да, не вопрос